### ISSUE TITLE
[Bug]: Blocked Apps gets updated

### ISSUE THREAD

**Issue by unclesam87** [Romanitho/Winget-AutoUpdate#479]
### The problem

hey,
i added after the install to the excluded_apps.txt some entrys (mozilla.thunderbird) to block them from update, but it seems like they still get updated - what is an big problem bc thunderbird tends to update to beta over winget.
Any ideas why the block list is ignored?
edit: just found an entry for mozilla.thunderbird in the winget_system_apps.txt - is there an problem maybe?

### What version of WAU has the issue?

1.18.1

### What version of Windows are you using (ex. Windows 11 22H2)?

Windows 10 to Windows 11 22H2

### What version of winget are you using?

1.6.3133

### Log information

```txt
Updating Mozilla Thunderbird (x64 de) from 115.3.3 to 115.4.1...
08:57:56 - ##########   WINGET UPGRADE PROCESS STARTS FOR APPLICATION ID 'Mozilla.Thunderbird.Beta'   ##########
08:57:56 - -> Running: Winget upgrade --id Mozilla.Thunderbird.Beta --accept-package-agreements --accept-source-agreements -h
```

**Comment by Romanitho** [comment]
Hi,

You said you've put `mozilla.thunderbird` in you black list. In the log, `Mozilla.Thunderbird.Beta` is updating.
Simply add `Mozilla.Thunderbird.Beta` to your black list too.

**Comment by unclesam87** [comment]
```
19:18:22 - Checking application updates on Winget Repository...
-> Available update : Mozilla Thunderbird (x64 de). Current version : 115.3.3. Available version : 115.4.1.
-> Available update : Mozilla Firefox (x64 de). Current version : 118.0.2. Available version : 119.0.
19:18:26 - Updating Mozilla Thunderbird (x64 de) from 115.3.3 to 115.4.1...
19:18:26 - ##########   WINGET UPGRADE PROCESS STARTS FOR APPLICATION ID 'Mozilla.Thunderbird.Beta'   ##########
19:18:27 - -> Running: Winget upgrade --id Mozilla.Thunderbird.Beta --accept-package-agreements --accept-source-agreements -h
```
look this is some kind of strange behavoiur - it seems like he pulls thunderbird an then switches to beta - or did i read the logs wrong?
